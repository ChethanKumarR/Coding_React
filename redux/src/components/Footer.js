import React from "react";


const Footer=()=>{
    return(
        <header>
            <div>
               <hr/>
               &copy; Developer Chethan
            </div>
        </header>
    )
}
export default Footer ;