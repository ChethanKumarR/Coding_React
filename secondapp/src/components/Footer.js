import React from "react";


const Footer = () => {
    return (
        <div>
            <hr />
            <center>
                <h2>&copy;React 2020</h2>
            </center>
        </div>
    )
}
export default Footer;  