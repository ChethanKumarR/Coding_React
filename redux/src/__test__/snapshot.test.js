import React from "react";
//importing create becoz we are doing functional or snapshot testing
import {create} from "react-test-renderer";
import Footer from "../components/Footer";

describe("Functional Testing", () => {
    //snapshot testing
    test("snapshotTtesting", () => {
        let tree = create(<Footer/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})