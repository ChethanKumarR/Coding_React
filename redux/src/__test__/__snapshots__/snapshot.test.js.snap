// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Functional Testing snapshotTtesting 1`] = `
<header>
  <div>
    <hr />
    Â© Developer Chethan
  </div>
</header>
`;
